CHANGELOG

```diff
@@ -2,6 +2,34 @@
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.4.3] - 2023-06-03
+
+### Added
+- Send quality stats to prometheus. (#1708)
+- Support for disabling publishing codec on specific devices (#1728)
+- Add support for bypass_transcoding field in ingress (#1741)
+- Include await_start_signal for Web Egress (#1759)
+
+### Fixed
+- Handle time stamp increment across mute for A/V sync (#1705)
+- Additional A/V sync improvements (#1712 #1724 #1737 #1738 #1764)
+- Check egress status on UpdateStream failure (#1716)
+- Start signal relay sessions with the correct node (#1721)
+- Fix unwrap for out-of-order packet (#1729)
+- Fix dynacast for svc codec (#1742)
+- Return max spatial layer from selectors (#1743)
```

Comment:
:nit: not a big deal, this one is not a fix, it is a change to aid the previous issue (dynacast for SVC)